terraform/executeTerraformTest.sh

```diff
@@ -63,7 +74,7 @@ esac
 
 test_framework_shortsha=$(git rev-parse --short HEAD)
 checkCache(){
-    CACHE_HIT=$(aws dynamodb get-item --region=us-west-2 --table-name ${DDB_TABLE_NAME} --key {\"TestId\":{\"S\":\"$1$2$3\"}\,\"aoc_version\":{\"S\":\"${TF_VAR_aoc_version}$test_framework_shortsha\"}})
```

Comment:
can't we use the other script `checkCacheHit` here instead of duplicating the logic?

Reply:
Done